src/test/java/uk/gov/digital/ho/hocs/workflow/processes/to/TO.java

```diff
@@ -204,6 +209,146 @@ public void testAllCloseAfterTriage() {
         verify(bpmnService).completeCase(any());
     }
 
+    @Test
+    public void TestCCHReturnsClose() {
+        whenAtCallActivity(DATA_INPUT)
+                .deploy(rule);
+
+        whenAtCallActivity(TRIAGE)
+                .alwaysReturn("BusAreaStatus", "Transferred")
+                .alwaysReturn("TROFTeamUUID", "b388a232-3f17-4f1e-8d86-30862b526d91")
+                .alwaysReturn("TriageOutcome", SEND_TO_DRAFT)
+                .deploy(rule);
+
+        whenAtCallActivity(EARLY_CLOSURE)
+                .deploy(rule);
+
+        whenAtCallActivity(CCH_RETURNS)
+                .alwaysReturn("BusAreaStatus", "Confirmed")
+                .alwaysReturn("CCHReturnsDirection", CCH_RETURNS_CLOSE)
+                .deploy(rule);
+
+        Scenario.run(TOProcess)
+                .startByKey("TO")
+                .execute();
+
+        verify(TOProcess, times(1))
+                .hasCompleted(START);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(DATA_INPUT);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(TRIAGE);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(CCH_RETURNS);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(EARLY_CLOSURE);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(COMPLETE_CASE);
+
+        verify(TOProcess, times(1))
+                .hasCompleted(END);
+
+        // NOT CALLED CHECK
+        verify(TOProcess, times(0))
+                .hasCompleted(CAMPAIGN);
+
+        verify(TOProcess, times(0))
+                .hasCompleted(STOP_LIST);
+
+        verify(TOProcess, times(0))
+                .hasCompleted(DRAFT);
+
+        verify(TOProcess, times(0))
+                .hasCompleted(QA);
+
+        verify(TOProcess, times(0))
+                .hasCompleted(DISPATCH);
+
+        // CASE COMPLETED
+        verify(bpmnService).completeCase(any());
+    }
+
+    @Test
+    public void testTransferFromCCHTriage(){
```

Comment:
@LaurieBennett can you add tests for transfer at Draft and QA please? I don't mind if you combine them with this, or have separate tests, I'd just like them covered.

Reply:
Added